ARCs/arc-0007.md

```diff
@@ -28,43 +28,43 @@ This ARC uses interchangeably the terms "throw an error" and "reject a promise w
 ### Interface `PostTxnsFunction`
 
 ```ts
-export type TxnId = string;
+export type TxnID = string;
 export type SignedTxnStr = string;
 
 export type PostTxnsFunction = (
   stxns: SignedTxnStr[],
 ) => Promise<PostTxnsResult>;
 
 export interface PostTxnsResult {
-  txnIds: TxnId[];
+  txnIDs: TxnID[];
 }
 
 export interface PostTxnsError extends Error {
   code: number;
   data?: any;
-  successTxnIds: (TxnId | null)[];
+  successTxnIDs: (TxnID | null)[];
 }
 ```
 
 A `PostTxnsFunction` with input argument `stxns:string[]` and promised return value `ret:PostTxnsResult`:
 * expects `stxns` to be in the correct string format as specified by `SignedTxnStr`.
-* **MUST**, if successful, return an object `ret` such that `ret.txId` is in the correct string format as specified by `TxId`.
+* **MUST**, if successful, return an object `ret` such that `ret.txID` is in the correct string format as specified by `TxID`.
```

Comment:
I think it is `txnIDs`?

Reply:
I'm not sure which is preferable but go-algorand has this as TxID

https://github.com/algorand/go-algorand/blob/bd18d0497f4a89b28fdd54811f95444adde901a4/daemon/algod/api/server/v2/generated/types.go#L704

Reply:
I have no strong opinions on which it should be, but the doc should at least be consistent with itself.